packages/runtime/src/useCases/common/RuntimeErrors.ts

```diff
@@ -140,29 +140,29 @@ class Notifications {
 }
 
 class Attributes {
-    public isNoIdentityAttribute(attributeId: CoreId | string): ApplicationError {
-        return new ApplicationError("error.runtime.attributes.isNoIdentityAttribute", `Attribute '${attributeId.toString()}' is not an identity attribute.`);
+    public isNotRepositoryAttribute(attributeId: CoreId | string): ApplicationError {
```

Comment:
I intentionally changed isNo to isNot, since from my point of view this is the correct negation. Moreover, we already have the check `isRepositoryAttribute()`, so for consistency I wouldn't add an article here.